libraries/ms-common/rules.mk

```diff
@@ -8,5 +8,5 @@
 $(T)_DEPS := $(PLATFORM_LIB) libcore
 
 ifeq (x86,$(PLATFORM))
-$(T)_EXCLUDE_TESTS := can can_hw adc
+$(T)_EXCLUDE_TESTS := can can_hw adc pwm
```

Comment:
Shame this list keeps getting longer. I'd really like to develop some infrastructure for proper input/output on x86. SocketCAN is still on my list of TODOs.